### ISSUE TITLE
Keras ShuffleNet v2 error conversion from onnx

### ISSUE THREAD

**Issue by xellDart** [Tencent/ncnn#3599]
## error log | 日志或报错信息 | ログ

Unsupported transpose type !
Unsupported transpose type !
Unsupported transpose type !
Unsupported transpose type !
Unsupported transpose type !
Unsupported transpose type !
Unsupported transpose type !
Unsupported transpose type !
Unsupported transpose type !
Unsupported transpose type !
Unsupported transpose type !
Unsupported transpose type !
Unsupported transpose type !
Unsupported transpose type !
Unsupported transpose type !

## model | 模型 | モデル
1. original model
Keras h5 https://app.nebuia.us-nyc1.upcloudobjects.com/model.h5
ONNX https://app.nebuia.us-nyc1.upcloudobjects.com/ShuffleNetV2_1.0_1_373.onnx

## how to reproduce | 复现步骤 | 再現方法
1. Convert keras o onnx: 

```
import tf2onnx
import onnxruntime as rt

spec = (tf.TensorSpec((None, 320, 320, 3), tf.float32, name="input"),)
output_path = model.name + ".onnx"

model_proto, _ = tf2onnx.convert.from_keras(model, input_signature=spec, opset=13, output_path=output_path)
output_names = [n.name for n in model_proto.graph.output]
```

3. Convert to onnx /onnx2ncnn ShuffleNetV2_1.0_1_373.onnx  model.param model.bin

Original post: https://www.kaggle.com/dcosmin/shufflenet-with-keras

I try with this mean norm values, this is correct?

 const float norm_vals[3] = {1/127.5f, 1/127.5f, 1/127.5f};
  in.substract_mean_normalize(0, norm_vals);

Thanks

**Comment by nihui** [comment]
try https://github.com/MarsTechHAN/keras2ncnn

**Comment by xellDart** [comment]
I already try this, but I get core dump error :(

**Comment by nihui** [comment]
针对onnx模型转换的各种问题，推荐使用最新的pnnx工具转换到ncnn
In view of various problems in onnx model conversion, it is recommended to use the latest pnnx tool to convert your model to ncnn

```shell
pip install pnnx
pnnx model.onnx inputshape=[1,3,224,224]
```

详细参考文档
Detailed reference documentation
https://github.com/pnnx/pnnx
https://github.com/Tencent/ncnn/wiki/use-ncnn-with-pytorch-or-onnx#how-to-use-pnnx
